lectures/equalizing_difference.md

```diff
@@ -447,44 +388,190 @@ This is because college graduates now have longer career lengths to "pay off" th
 Let's compute $\frac{\partial \phi}{\partial γ_h}$ and evaluate it at default parameters.
 
 ```{code-cell} ipython3
-ϕ_γ_h = ϕ(D, γ_h, γ_c, R, T, w_h0).diff(γ_h)
+ϕ_γ_h = ϕ.diff(γ_h)
 ϕ_γ_h
 ```
 
 ```{code-cell} ipython3
 # Numerical value at default parameters
-ϕ_γ_h_func = Lambda((D, γ_h, γ_c, R, T, w_h0), ϕ_γ_h)
-ϕ_γ_h_func(D_value, γ_h_value, γ_c_value, R_value, T_value, w_h0_value)
+ϕ_γ_h.subs(symbol_subs)
 ```
 
 We find that raising $\gamma_h$ increases the initial college wage premium $\phi$, in line with our earlier graphical analysis.
 
 Compute $\frac{\partial \phi}{\partial γ_c}$ and evaluate it numerically at default parameter values
 
 ```{code-cell} ipython3
-ϕ_γ_c = ϕ(D, γ_h, γ_c, R, T, w_h0).diff(γ_c)
+ϕ_γ_c = ϕ.diff(γ_c)
 ϕ_γ_c
 ```
 
 ```{code-cell} ipython3
 # Numerical value at default parameters
-ϕ_γ_c_func = Lambda((D, γ_h, γ_c, R, T, w_h0), ϕ_γ_c)
-ϕ_γ_c_func(D_value, γ_h_value, γ_c_value, R_value, T_value, w_h0_value)
+ϕ_γ_c.subs(symbol_subs)
 ```
 
 We find that raising $\gamma_c$ decreases the initial college wage premium $\phi$, in line with our earlier graphical analysis.
 
 Let's compute $\frac{\partial \phi}{\partial R}$ and evaluate it numerically at default parameter values
 
 ```{code-cell} ipython3
-ϕ_R = ϕ(D, γ_h, γ_c, R, T, w_h0).diff(R)
+ϕ_R = ϕ.diff(R)
 ϕ_R
 ```
 
 ```{code-cell} ipython3
 # Numerical value at default parameters
-ϕ_R_func = Lambda((D, γ_h, γ_c, R, T, w_h0), ϕ_R)
-ϕ_R_func(D_value, γ_h_value, γ_c_value, R_value, T_value, w_h0_value)
+ϕ_R.subs(symbol_subs)
 ```
 
 We find that raising the gross interest rate $R$ increases the initial college wage premium $\phi$, in line with our earlier graphical analysis.
+
+## Exercises
+
+In the following exercises, we extend our previous model to a model of entrepreneurs versus workers. 
```

Comment:
@SylviaZhaooo @HumphreyYang I am not really sure what this sentence means with regards `entrepreneurs` vs `workers`. Is it a model of choice between being an entrepreneur vs a worker? Can this be explained better.